VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.ts

```diff
@@ -85,27 +85,21 @@ export class JudgeContextMenuComponent {
         this.isDroppedDown = !this.isDroppedDown;
     }
 
-    getAdditionalText(): string {
-        return this.participant.hearingRole !== HearingRole.JUDGE ? this.getHearingRole() + this.getCaseRole() : '';
+    get isJudge(): boolean {
+        return this.participant.hearingRole === HearingRole.JUDGE;
     }
 
-    private getHearingRole(): string {
-        return this.participant.representee
-            ? `<br/>${this.participant.hearingRole} for ${this.participant.representee}`
-            : `<br/>${this.participant.hearingRole}`;
+    get isWitness(): boolean {
+        return this.participant.hearingRole === HearingRole.WITNESS;
     }
 
-    private getCaseRole(): string {
-        return this.showCaseRole() ? `<br/>${this.participant.caseTypeGroup}` : '';
-    }
-
-    private showCaseRole() {
+    showCaseTypeGroup(): boolean {
         return !this.participant.caseTypeGroup ||
             this.participant.caseTypeGroup.toLowerCase() === CaseTypeGroup.NONE.toLowerCase() ||
             this.participant.caseTypeGroup.toLowerCase() === CaseTypeGroup.OBSERVER.toLowerCase() ||
             this.participant.caseTypeGroup.toLowerCase() === CaseTypeGroup.PANEL_MEMBER.toLowerCase() ||
             this.participant.caseTypeGroup.toLowerCase() === CaseTypeGroup.JUDGE.toLowerCase() ||
-            this.participant.caseTypeGroup.toLowerCase() === 'endpoint'
+            this.participant.caseTypeGroup.toLowerCase() === CaseTypeGroup.ENDPOINT.toLowerCase()
```

Comment:
This should also include staff member if we are handling that. @krishnapottekkat-hmcts @alberto-hortiguela @simonwkelso